content/updates/2018-02.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 2018.2.0
```

Comment:
The title here should be 'March 2018' to match the other one. Now version numbers are formalised, i'm going to add an additional column over the weekend to track this

Reply:
See #127

Reply:
I think we should call this the "Vision update", as it's how we're referring to it.

Reply:
fixed in ae90ce899c9aefea6fea595aa44711d0f36ae58a as I don'y like my  'vision update' suggestion any mre